spec/_attachments/ic.did

```diff
@@ -342,6 +342,14 @@ type node_metrics_history_result = vec record {
     node_metrics : vec node_metrics;
 };
 
+type subnet_stats_args = record {
```

Comment:
For consistency with the node metrics and the subnet metrics in the state tree – which may eventually converge to contain the same information – should we call this `subnet_metrics_...`?

Reply:
Sounds good, I'll change it. Unless you are waiting for an opinion from somebody else?